server/blindai_sgx/src/lib.rs

```diff
@@ -146,6 +147,30 @@ async fn main(
         network_config.max_input_size,
     );
 
+    let mut models_path = std::env::current_dir()?;
+    models_path.push("models");
+    if let Ok(paths) = fs::read_dir(models_path.as_path()) {
```

Comment:
all of this should ideally be in the `model_store.rs` and `sealing.rs` file ; Separation of concerns & avoid redundant code

Maybe add a `model_store::startup_unseal` function to unseal every models from folder.